README.md

```diff
@@ -50,31 +50,22 @@ the central point of interaction for users:
   an update mechanism so there will only need to be a single store xdc app in
   the store bot chat. 
 
-## Setting Up the Bot
+## Setting up the bot
 
-### Requirements
-
-You need to have `rust` and `node` installed.
-Optionally, have [pnpm](https://pnpm.io/) installed.
-
-### Setting up the bot 
+### Installation
 
-Go to the `frontend` folder and install the packages using `pnpm install` or `pnpm i`.
-If you do not have `pnpm` installed, you can run `npx pnpm install` to use `pnpm` 
-without installing it globally or use `npm install` instead.
-However, if you use `npm install`, the latest versions of dependencies will be installed
-instead of the ones listed in the `frontend/pnpm-lock.yaml` file.
+TODO: Download the latest release from https://github.com/webxdc/store/releases,
+extract the compressed archive to the desired location, it containts the bot command line
+program and required assets.
 
-Build the frontend by running `npm run build` or `pnpm run build`.
-This step creates a `bot-data` directory in the root of the repository
-with files `appstore.xdc`, `review-helpr.xdc` and `submit-helper.xdc`.
+### Running the bot
 
 To run the bot, set the environment variables
 `addr` and `mail_pw` with the login and password
 and use the command `start`:
 
 ```
-    addr=bot@example.org mail_pw=My_P4ssword cargo run -- start
+    addr=bot@example.org mail_pw=My_P4ssword ./xdcstore start
```

Comment:
this will change based on the resolution of https://github.com/webxdc/store/issues/89